### PR TITLE
Create print method for `camtrapdp` objects

### PR DESCRIPTION
**Pull request by PietrH**
This PR implements a way to do this:

```r
print(example_dataset())
```

It's a slight extension on what frictionless does already, but also counts rows of the resources included in `x$data`, and lists any non standard resources in `x$resources` with a note that they are non standard. 

I've created an issue #90 where I've pasted a snippet on how we could go about counting non tabular custom resources. But I feel this code should go into `read_camtrapdp()` rather than this print method. So that's why I abandoned work in it for now. 

I'd be very interested in your views on the tests. I'm not super happy about them. Maybe it's better to create a camtrapdp from scratch with frictionless and test more cases on that? Maybe it's to strict already..

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,6 @@
 Package: camtrapdp
 Title: Read and Manipulate Camera Trap Data Packages
-Version: 0.2.1.9000
-Date: 2024-06-05
+Version: 0.2.2.9000
```
Comment (peterdesmet) [inline DESCRIPTION:3]: Please revert to 0.2.1.9000. The progression in version numbers is:

- 0.2.1 (latest CRAN release)
- 0.2.1.9000 (current version number + 9000): ongoing work
- whatever version we decide to release next (major, minor, patch)

File: DESCRIPTION [hunk 2]
```diff
@@ -1,7 +1,6 @@
 Package: camtrapdp
 Title: Read and Manipulate Camera Trap Data Packages
-Version: 0.2.1.9000
-Date: 2024-06-05
```
Comment (peterdesmet) [inline DESCRIPTION:4]: I know it's annoying (and against pkg recommendations), but please keep the `Date` property. It is required for a bunch of the generated metadata like cff

File: NEWS.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # camtrapdp (development version)
 
+# camtrapdp 0.2.2
```
Comment (peterdesmet) [inline NEWS.md:3]: Don't add this header yet. Any current development goes under `camtrapdp (development version)`

**Review by peterdesmet — CHANGES_REQUESTED** [review]
I first cursory review, didn't test actual functionality yet.

File: DESCRIPTION (hunk 2 above)
Comment (PietrH) [reply DESCRIPTION:4]: What date would you like me to put here?

**Comment by PietrH** [comment]
@peterdesmet ready for review

**Review by peterdesmet — APPROVED** [review]
I reworked the print, so it now producues:

```
A Camera Trap Data Package with 3 tables:
• deployments: 4 rows
• media: 423 rows
• observations: 549 rows

And 1 additional resource, which can be loaded with `frictionless::read_resource()`:
• individuals
Use `unclass()` to print the Data Package as a list.
```

I have simplified the tests, it's fine they're a bit hardcoded, since we know what to expect in the example_dataset.

Please have a look and merge if fine.

**Comment by PietrH** [comment]
Looks good, but `frictionless::read_resource()` can not read `individuals`

frictionless::read_resource(camtrapdp::example_dataset(), "individuals")
#> Error in `get_schema()`:
#> ! Resource "individuals" must have a profile property with value
#>   "tabular-data-resource".

**Comment by peterdesmet** [comment]
Right, maybe we should just advice to use the frictionless package (not a specific function)? The individuals resource is an odd one (json), not (yet) supported by frictionless. Most will attach csv data.

**Comment by peterdesmet** [comment]
I won't mention frictionless in print.

**Pull request merged**